sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/RequestResponseModelTests.cs

```diff
@@ -23,13 +23,13 @@ public void ConcurrencyTest()
             for (int i = 0; i < 10000; i++)
                 ThreadPool.QueueUserWorkItem(async w =>
                 {
-                    HttpResponseMessage httpResponseMessage = await TestHelper.EventResponseBaseTest(eventsResponseHandler =>
+                    HttpResponseMessage httpResponseMessage = await EventResponseBaseTest(async eventsResponseHandler =>
                     {
-                        eventsResponseHandler.SetValueAsync(Payload.TokenIssuanceStart.ActionResponse, CancellationToken.None);
+                        await eventsResponseHandler.SetValueAsync(Payload.TokenIssuanceStart.ActionResponse, CancellationToken.None);
                     });
 
-                    Assert.AreEqual(System.Net.HttpStatusCode.OK, httpResponseMessage.StatusCode);
-                    Assert.True(TestHelper.DoesPayloadMatch(Payload.TokenIssuanceStart.ExpectedPayload, httpResponseMessage.Content.ReadAsStringAsync().Result));
+                    Assert.AreEqual(HttpStatusCode.OK, httpResponseMessage.StatusCode);
+                    Assert.True(DoesPayloadMatch(Payload.TokenIssuanceStart.ExpectedPayload, httpResponseMessage.Content.ReadAsStringAsync().Result));
```

Comment:
Please await on `ReadAsStringAsync()` before using it in assert

Reply:
Done

Reply:
I have updated the tests that use ReadAsStringAsync to be awaited. And I will create an ADO work item to update them in `AuthenticationEventResponseHandler.cs` and `AuthenticationEventResponse.cs`